packages/protoc-gen-connect-query-react/README.md

```diff
@@ -1,5 +1,7 @@
 # @bufbuild/protoc-gen-connect-query-react
 
+**This is an experimental plugin, based on the [protoc-gen-connect-query](../protoc-gen-connect-query/README.md) plugin. This plugin provides better DX while sacrificing some flexibility.**
```

Comment:
Can we point out the main difference to protoc-gen-connect-query? It generates hooks for react.

We can't link to `../protoc-gen-connect-query/README.md` here because that doesn't resolve on npmjs.com. Let's link to https://www.npmjs.com/package/@bufbuild/protoc-gen-connect-query instead.